tests/test_image.py

```diff
@@ -35,3 +35,10 @@ def test_com_sparse(sparse_array_small, full_array_small):
             full_com[:, i, j] = com_dense(full_array_small[i, j])[:, 0]
 
     assert np.array_equal(com, full_com)
+
+
+def test_radial_sum_sparse(sparse_array_10x10):
+
+    rr = radial_sum_sparse(sparse_array_10x10, center=(4, 4))
+
+    assert np.all(rr[0, 0, :] == [0, 6, 0, 0, 3])
```

Comment:
It appears that this is failing in the tests, where `rr[0, 0, :]` is actually coming out to be `[3, 0, 3, 0, 0]` instead. @ercius any idea of what might be wrong?

Note that using `np.array_equal()` might be a little faster as well: `np.array_equal(rr[0, 0, :], [0, 6, 0, 0, 3])`

Reply:
My test is incorrect. The line should be:
`rr = radial_sum_sparse(sparse_array_10x10, center=(5, 5))`
and then the output `[0, 6, 0, 0, 3]` is correct.

Ill also update to using `np.array_equal()`